library/bignum_mod_raw.c

```diff
@@ -120,6 +120,16 @@ void mbedtls_mpi_mod_raw_sub( mbedtls_mpi_uint *X,
     (void) mbedtls_mpi_core_add_if( X, N->p, N->limbs, (unsigned) c );
 }
 
+void mbedtls_mpi_mod_raw_mul( mbedtls_mpi_uint *X,
```

Comment:
Why is that method required if it is only an alias for the core one? I would remove it, and invoke the core method directly. It will reduce code size and remove duplicate testing requirements

Reply:
Actually, this is already in `development`, merged as part of #6742 which you approved :)

The difference is that the function in `mod_raw` takes a `mbedtls_mpi_mod_modulus *` but the function in core just takes `mbedtls_mpi_uint *`s